prometheus-to-sd/config/dynamic_source.go

```diff
@@ -31,11 +52,13 @@ func SourceConfigsFromDynamicSources(gceConfig *GceConfig, sources []flags.Uri)
 	if err != nil {
 		return nil, err
 	}
-	podResponse, err := kubeApi.CoreV1().Pods(podNamespace).List(createOptionsForPodSelection(gceConfig.Instance, sourceMap))
+	podApi := kubeApi.CoreV1().Pods(podNamespace)
+	podOptions := createOptionsForPodSelection(gceConfig.Instance, sourceMap)
+	watchInterface, err := podApi.Watch(podOptions)
```

Comment:
I think creating watch interface should be moved to `getSourceConfigChangeChannel` function

Reply:
done